### PR TITLE
Hardware / Sim Dependencies

### PR DESCRIPTION
**Pull request by rafal-gorecki**

### CODE REVIEW

File: panther_bringup/package.xml [hunk 1]
```diff
@@ -15,13 +15,13 @@
...
-  <exec_depend>imu_filter_madgwick</exec_depend>
+  <exec_depend condition="$HUSARION_ROS_BUILD_TYPE == hardware">imu_filter_madgwick</exec_depend>
   <exec_depend>launch</exec_depend>
   <exec_depend>launch_ros</exec_depend>
-  <exec_depend>panther_battery</exec_depend>
+  <exec_depend condition="$HUSARION_ROS_BUILD_TYPE == hardware">panther_battery</exec_depend>
   <exec_depend>panther_controller</exec_depend>
-  <exec_depend>panther_lights</exec_depend>
-  <exec_depend>phidgets_spatial</exec_depend>
+  <exec_depend condition="$HUSARION_ROS_BUILD_TYPE == hardware">panther_lights</exec_depend>
```
Comment (KmakD) [inline panther_bringup/package.xml]:
```suggestion
  <exec_depend>panther_lights</exec_depend>
```
Simulation may want to use lights

**Review by KmakD — CHANGES_REQUESTED** [review]

File: panther_bringup/launch/bringup.launch.py [hunk 2]
```diff
@@ -256,11 +256,11 @@ def generate_launch_description():
         PythonLaunchDescriptionSource(
             PathJoinSubstitution(
                 [
-                    get_package_share_directory("panther_battery"),
+                    # FindPackageShare("panther_battery"),
```
Comment (rafal-gorecki) [inline panther_bringup/launch/bringup.launch.py]:
```suggestion
                    FindPackageShare("panther_battery"),
```

File: panther_controller/README.md [hunk 3]
```diff
@@ -1 +1,60 @@
...
+
+```bash
+ros2 launch panther_bringup bringup.launch.py
+```
+
+Simulation:
+
+```bash
+ros2 launch panther_gazebo simulation.launch.py
+```
```
Comment (KmakD) [inline panther_controller/README.md]: I think this better fits main repository README than panther_controller one

**Review by KmakD — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -4,6 +4,66 @@ ROS 2 packages for Panther autonomous mobile robot
...
+rosdep install --from-paths src -y -i
+
+source /opt/ros/$ROS_DISTRO/setup.bash
+colcon build --symlink-install --packages-up-to panther --cmake-args -DCMAKE_BUILD_TYPE=Release
+
+source install/setup.bash
+```
+
+>[!NOTE]
+> To build code on real robot you need to running above commands on the Panther Built-in Computer.
```
Comment (KmakD) [inline README.md]:
```suggestion
> To build code on a real robot you need to run above commands on the Panther Built-in Computer.
```

**Review by KmakD — CHANGES_REQUESTED** [review]

**Review by KmakD — APPROVED** [review]

**Pull request merged**